survey/survey-service-offering_shacl.ttl

```diff
@@ -0,0 +1,75 @@
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix survey: <http://semanticweb.org/metadatasurveylogy/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
+
+survey:SurveyServiceOfferingShape
+	a sh:NodeShape ; 
+	sh:targetClass survey:SurveyServiceOffering ; 
+	sh:property [
+		sh:path survey:surveyId ; 
```

Comment:
you can leave this out because you will have a `id` field which must match the ID of the software resource (see comment in ontology and OpenAPI-Description in https://claim-compliance-provider.gxfs.gx4fm.org/docs/#/Claim%20Compliance%20Provider%20Controller/post_v1_send_claims or updated description in https://github.com/GAIA-X4PLC-AAD/gaia-x-compliant-claims-example/pull/9)